en/docs/assets/attachments/sample-sso-redirect-page-template.html

```diff
@@ -0,0 +1,41 @@
+<!-- 
+Variables $acUrl, $response, $relayState and $additionalParams will be replaced by the corrosponding values  
+-->
+<html>
```

Comment:
Fix indentations in the file. Use 4 spaces.

Reply:
Fixed